### PR TITLE
Project Plan 4.34

### PR DESCRIPTION
**Pull request by elsazac**
Fixes: https://github.com/eclipse-platform/www.eclipse.org-eclipse/issues/229

Created Project Plan for 4.34

### CODE REVIEW

**Comment by elsazac** [comment]
The build looks okay, so I am planning to merge this now. 

@akurtakov Could you kindly take a quick look at this and let me know if any corrections are needed?

**Comment by akurtakov** [comment]
Please always use latest Java versions e.g. 17.0.12 and 21.0.4 are out and we should always aim at keeping this list uptodate as old versions are not supported.

File: development/plans/eclipse_project_plan_4_34.xml [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+			<td rowspan="1">Power 64-bit LE</td>
+			<td rowspan="1">
+				OpenJDK 17.0.11 (LTS)<br/>
+			</td>
+		</tr>
+		<!-- ************ SLES ************** -->
+		<tr class="c0">
+			<td rowspan="2">SUSE Linux Enterprise Server</td>
+			<td rowspan="2">
+				15 SP4
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_34.xml:228]: Update to 15 SP6 as this is the latest version AFAICT.

File: development/plans/eclipse_project_plan_4_34.xml [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+				Oracle Java 17.0.11 (LTS)<br/>
+				Oracle Java 21.0.2 (LTS)<br/>
+			</td>
+			<td rowspan="1">Win32</td>
+		</tr>
+		<!-- ************ RHEL ************** -->
+		<tr class="c1">
+			<td rowspan="2">Red Hat Enterprise Linux</td>
+
+			<td rowspan="2">9.0
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_34.xml:204]: Update to 9.4 - latest RHEL

File: development/plans/eclipse_project_plan_4_34.xml (hunk 1 above)
Comment (elsazac) [reply development/plans/eclipse_project_plan_4_34.xml]: Done.

File: development/plans/eclipse_project_plan_4_34.xml (hunk 2 above)
Comment (elsazac) [reply development/plans/eclipse_project_plan_4_34.xml]: Done

**Comment by elsazac** [comment]
@akurtakov Thanks for the review.

**Comment by elsazac** [comment]
Build looks good.
I am merging this now.

**Pull request merged**